pixeltable/catalog/catalog.py

```diff
@@ -513,9 +541,98 @@ def _acquire_tbl_lock(
                 )
         return TableVersionHandle(tbl_id, effective_version)
 
+    def _finalize_pending_ops(self, tbl_id: UUID) -> None:
+        """
+        Finalizes the next pending op for the given table. Returns True if there was a pending op, False otherwise.
```

Comment:
Docstring talks about returning `True`/`False`, but return type is `None`

Reply:
Done.